app/components/carousel/carousel.tsx

```diff
@@ -0,0 +1,98 @@
+'use client'
+
+import React from 'react'
+import Slider from 'react-slick'
+import { Box, IconButton, useBreakpointValue } from '@chakra-ui/react'
+import { BiLeftArrowAlt, BiRightArrowAlt } from 'react-icons/bi'
+
+const settings = {
+  dots: true,
+  arrows: false,
+  fade: true,
+  infinite: true,
+  autoplay: true,
+  speed: 500,
+  autoplaySpeed: 5000,
+  slidesToShow: 1,
+  slidesToScroll: 1
+}
+
+export const Carousel = () => {
```

Comment:
I think it's better not to use auto change mode. Because it's not for marketing stuff! And if you have button inside the image, you never be able to see the part of image so could you change the place as design or outside of the image? 🙏